ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/Genesis/Setup/GenChains.hs

```diff
@@ -116,6 +116,7 @@ genChains genNumForks = do
     gtHonestAsc = asc,
     gtSecurityParam = SecurityParam (fromIntegral kcp),
     gtGenesisWindow = GenesisWindow (fromIntegral scg),
+    gtForecastRange = ForecastRange (fromIntegral scg), -- REVIEW: Do we want to generate those randomly?
```

Comment:
I think we always want to set it to something equal to or greater than the Genesis window for tests that are expected to succeed. It might be interesting to set it to smaller values to ensure we have good test coverage as there should be cases where this causes a failure (liveness).

Reply:
Related: https://github.com/IntersectMBO/ouroboros-consensus/commit/4466b724dd324e11e9eafc970d010a3b942655f3.